ts/Grid/Core/Credits.ts

```diff
@@ -200,7 +163,25 @@ class Credits {
  * */
 
 namespace Credits {
```

Comment:
I would change the name of the class & namespace from `Credits` to e.g. `CreditsLite`.